src/sentry/static/sentry/app/api.jsx

```diff
@@ -174,11 +174,19 @@ export class Client {
       });
       throw err;
     }
+
     const method = options.method || (options.data ? 'POST' : 'GET');
     let data = options.data;
     const id = uniqueId();
     metric.mark(`api-request-start-${id}`);
 
+    const hub = Sentry.getCurrentHub();
+    const requestSpan = hub.startSpan({
```

Comment:
@HazAT it'll be a good idea to move this all the way to the beginning of this function. In case this line `query = $.param(options.query || [], true);` borks again. see: https://github.com/getsentry/sentry/pull/14051